src/main/java/play/libs/mailer/Email.java

```diff
@@ -99,11 +99,16 @@ public Email setBcc(List<String> bcc) {
     return this;
   }
 
-  public String getReplyTo() {
+  public List<String> getReplyTo() {
```

Comment:
@Mogztter We also would need to deprecate this one and instead add something like `public List<String> getRepliesTo() { ... }` or `getAllReplyTo()`.
The question though is what the deprecated `String getReplyTo` should return? Just the first email in the list (if there is one)?